mibidata/panels.py

```diff
@@ -61,11 +62,16 @@ def read_csv(path):
 
 def _merge_masses(df):
     conjugates = {}
+    target_list = []
     for conj in df.to_dict(orient='records'):
         mass = conj['Mass']
         target = conj['Target']
         if conjugates.get(mass):
-            conjugates[mass] = f'{conjugates[mass]}, {target}'
+            conjugates[mass].append(target)
         else:
-            conjugates[mass] = target
+            conjugates[mass] = [target]
+    for mass in conjugates:
+        target_list = conjugates[mass]
+        util.natural_sort(target_list)
```

Comment:
I think natural sort is not really needed here, but it doesn't hurt to use it either.

Reply:
It's not necessary for enforcing the same order each time, yes, but it will make ordering consistent with other places in MIBItracker, mibitof, etc.

Reply:
I meant we don't have targets like Target1, Target2, ...... Target10. The standard function `conjugates[mass].sort()` could also work fine.